plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/LibraryClasspathContainer.java

```diff
@@ -86,7 +87,7 @@ public String getDescription() {
                                      true);
       }
       return entries;
```

Comment:
I think the repositoryService should only be consulted for the {source|javadoc} if a {sourceUri|javadocUri} wasn't provided, right?